src/main/java/pl/vishop/vishopplugin/config/Config.java

```diff
@@ -30,19 +31,19 @@ public class Config {
     public boolean load(final ConfigurationSection cfg) {
         this.apiKey = cfg.getString("apiKey", "");
         if (this.apiKey.isEmpty()) {
-            Bukkit.getLogger().warning("Uzupełnij klucz API w configu, aby łączyć się ze swoim sklepem w ViShop");
+            Bukkit.getLogger().warning(ChatColor.RED + "Musisz uzupełnić klucz API w pliku konfiguracyjnym, aby plugin działał. Wyłączam...");
```

Comment:
I w każdej wiadomości jest taki sam tekst?

Reply:
Błąd xD
Trzeci raz już to robiłem i mi się zapomniało